test-fixtures/src/bin/acl.rs

```diff
@@ -0,0 +1,62 @@
+//! A guest program to test that acls works properly.
+use fastly::Error;
+
+fn main() -> Result<(), Error> {
+    // Temporary until fastly SDK is released which
+    // includes the fastly::acl module.
+    #[cfg(feature = "acl_hostcalls")]
```

Comment:
If I'm following right, is the idea here to:

- Merge this PR
- Publish Viceroy
- Publish Rust SDK with ACLs support
- Do follow-up PR to remove the `acl_hostcalls` feature guard

Reply:
Yes, that's also how I was thinking the sequencing would be. 

This feature guard can be removed once the new `fastly` crate is published (the order of the previous steps shouldn't matter).